features/cubic-bezier-easing.yml

```
@@ -0,0 +1,6 @@
+name: cubic-bezier() easing
+description: The `cubic-bezier()` easing function interpolates from 0-1 along a smooth curve influenced by two control points, used to create animations and transitions with continuous changes in speed. The `ease`, `ease-in`, `ease-out`, and `ease-in-out` CSS properties are presets for common scenarios.
```

```suggestion
description: The `cubic-bezier()` easing function creates animations and transitions with continuous changes in speed. The `ease`, `ease-in`, `ease-out`, and `ease-in-out` CSS properties are presets for common scenarios.
```
Potentially, we could add `interpolates from 0-1 along a smooth curve influenced by two control points` to the end, but I'm not sure if it's a crucial part- I think the "continuous changes in speed" line explains it better (if less specifically).

---

I was following the pattern established by the `linear()` easing function here:
> The `linear()` easing function for animations and transitions interpolates linearly between the control points, and can be used to approximate complex easing functions, such as a bounce effect.

I'm not really a fan of "creates" here, but fewer internal details would probably help. Maybe something more like: 
> The `cubic-bezier()` easing function interpolates along a smooth curve used to create animations and transitions with continuous changes in speed. The `ease`, `ease-in`, `ease-out`, and `ease-in-out` CSS properties are presets for common scenarios.

---

I like your proposal! I think you can drop the word "used" here.